LuaUI/widgets/gui_standalone_lobby.lua

```diff
@@ -1,17 +1,20 @@
 function widget:GetInfo()
-  return {
-	name      = "Standalone lobby tools",
-	desc      = "Standalone lobby tools",
-	author    = "gajop",
-	date      = "in the future",
-	license   = "GPL-v2",
-	layer     = 1001,
-	enabled   = true,
-  }
+	return {
+		name      = "Standalone lobby tools",
+		desc      = "Standalone lobby tools",
+		author    = "gajop",
+		date      = "in the future",
+		license   = "GPL-v2",
+		layer     = 1001,
+		enabled   = true,
+	}
 end
 
+spSendCommands = Spring.SendCommands
```

Comment:
I deeply dislike localizing (Spring) function names like this. Write the whole thing.